lib/Event/DataObjectEvents.php

```diff
@@ -67,6 +67,13 @@
      */
     const POST_UPDATE_FAILURE = 'pimcore.dataobject.postUpdateFailure';
 
+     /**
+     * @Event("Pimcore\Event\Model\DataObjectEvent")
```

Comment:
maybe change to DataObjectDeleteInfoEvent  ?

Reply:
done